evalml/model_understanding/graphs.py

```diff
@@ -659,6 +659,14 @@ def partial_dependence(
             f"Columns {X_unknown.columns.values} are of type 'Unknown', which cannot be used for partial dependence"
         )
 
+    X_not_allowed = X_features.ww.select(["URL", "EmailAddress", "NaturalLanguage"])
+    if len(X_not_allowed.columns):
+        # these three logical types aren't allowed for partial dependence
```

Comment:
Nit picky but if we want to keep this comment, maybe annotate `X_not_allowed` instead? :')